tests/python/pytorch/nn/test_nn.py

```diff
@@ -1839,6 +1839,64 @@ def forward(self, g, h, embed=False, edge_weight=None):
     probs, edge_weight = explainer.explain_graph(g, feat)
 
 
+@pytest.mark.parametrize("g", get_cases(["hetero"], exclude=["zero-degree"]))
```

Comment:
Does this test cover batched graphs?

Reply:
sorry has me a bit confused, should I try to cover all possible heterographs in `get_cases`?

something like this:
```suggestion
@pytest.mark.parametrize("g", get_cases(["hetero", "bipartite", "block"], exclude=["block-bipartite"]))
```

Reply:
From the documentation you added, the module supports a batched graph, one that is created with [dgl.batch](https://docs.dgl.ai/generated/dgl.batch.html#dgl.batch). However, previously the unit test did not cover a batched heterogeneous graph. I was suggesting the expansion of the unit test to cover batched heterogeneous graphs. You don't necessarily need to use `get_cases`. It's fine to directly create graphs inside the unit test function.

Reply:
oh i understand now, sorry for the confusion. It makes sense to me to keep a case for this so let me try to make an addition to `get_cases`

Reply:
added